homeassistant/components/tradfri/cover.py

```diff
@@ -82,7 +82,7 @@ def name(self):
     def current_cover_position(self):
         """Return current position of cover.
 
-        None is unknown, 0 is closed, 100 is fully open.
+        None is unknown, 100 is closed, 0 is fully open.
```

Comment:
We can't change this definition. Home assistant should present a coherent interface to users. We have to invert the position we get from the device.

https://developers.home-assistant.io/docs/en/entity_cover.html#platform-properties-to-be-implemented-by-deriving-platform-classes